src/components/pokemonStatisticsModal.html

```diff
@@ -118,6 +119,15 @@ <h5 class="modal-title text-light" data-bind="if: App.game.party.getPokemon(poke
                       </td>
                     </tr>
                     <!-- /ko -->
+                    <tr>
+                        <td colspan="2" class="text-left align-middle">Party Nickname</td>
```

Comment:
I think it looks a little better with this text centered
```suggestion
                        <td colspan="2" class="text-middle align-middle">Party Nickname</td>
```